autospotting.go

```diff
@@ -26,7 +26,7 @@ var Version string
 var ExpirationDate string
 
 func main() {
-	run()
+	lambda.Start(Handler)
```

Comment:
I think we can use one of the [reserved environment variables](
https://docs.aws.amazon.com/lambda/latest/dg/current-supported-versions.html) to determine if the binary is executed in the Lambda environment so we then run the handler, and keep calling the `run()` when executed locally, like we do now. A simple if like this one should do the trick (note: this is just a hunch, and it wasn't yet tested): 

```
if os.Getenv("AWS_LAMBDA_FUNCTION_NAME") != "" {
  lambda.Start(Handler)
} else {
 run()
}
```

Reply:
Exactly my thought